cmd/hack/hack.go

```diff
@@ -2256,22 +2284,68 @@ func (r *Receiver) Result() trie.SubTries {
 	return r.defaultReceiver.Result()
 }
 
-func testGetProof(chaindata string, block uint64, account common.Address) {
-	fmt.Printf("testGetProof %s, %d, %x\n", chaindata, block, account)
-	// Find all changesets larger than given blocks
-	db, err := ethdb.NewBoltDatabase(chaindata)
-	check(err)
-	defer db.Close()
+func identityLoadFunc(k []byte, valueDecoder etl.Decoder, _ etl.State, next etl.LoadNextFunc) error {
```

Comment:
maybe we can move that to `etl` package, because it might be useful in a couple of places?

Reply:
Done, thank you